src/develop/kotlin/mocks/LoadData.kt

```diff
@@ -1,11 +1,26 @@
 package community.flock.eco.workday.mocks
 
-import community.flock.eco.workday.repository.DayRepository
+import community.flock.eco.feature.user.repositories.UserRepository
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
 import org.springframework.stereotype.Component
 
 @Component
 @ConditionalOnProperty(prefix = "flock.eco.workday", name = ["develop"])
 class LoadData(
-    val dayRepository: DayRepository
-)
+        private val userRepository: UserRepository
+) {
+    fun loadWhenEmpty(block: () -> Unit) {
+
+        if(load == null){
+            load = userRepository.count() == 0L
+        }
+
+        if (load == true) {
```

Comment:
whitespace betweeen closing paranthesis and bracket ') {'
is the code formatted correctly?